### PR TITLE
Added a link to resources

### PR DESCRIPTION
**Pull request by joinimran**
Added a link to the resources where the user can get more details related to permissions.

Problem: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/817

### CODE REVIEW

**Comment by joinimran** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/add-or-remove-members-from-a-case-in-advanced-ediscovery.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ description: ""
 ---
 # Add or remove members from a case
 
-You can add or remove members to manage who can access the case. Users who are added to Reviewer or eDiscovery Manager role groups can be added as members of the case. The eDiscovery Manager who created the case is automatically added as a member.
+You can add or remove members to manage who can access the case. Users who are added to Reviewer or eDiscovery Manager role groups can be added as members of the case. The eDiscovery Manager who created the case is automatically added as a member. See [assign eDiscovery permissions in the Security & Compliance Center](https://docs.microsoft.com/microsoft-365/compliance/assign-ediscovery-permissions) for more details.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/add-or-remove-members-from-a-case-in-advanced-ediscovery.md]:
```suggestion
You can add or remove members to manage who can access the case. Users who are added to Reviewer or eDiscovery Manager role groups can be added as members of the case. The eDiscovery Manager who created the case is automatically added as a member. See [Assign eDiscovery permissions in the Security & Compliance Center](https://docs.microsoft.com/microsoft-365/compliance/assign-ediscovery-permissions) for more details.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - minor edit.

**Comment by joinimran** [comment]
@mypil Ready to go live.

**Comment by mypil** [comment]
@markjjo - PR has been copy edited and is ready for final review. This is based on issue #817.

Thank you.

cc: @kenwith

**Pull request merged**